README.md

```diff
@@ -480,18 +480,19 @@ Similarly, cross-document navigations initiated from other windows are not somet
 
 As for `document.open()`, it is a terrible legacy API with lots of strange side effects, which makes supporting it not worth the implementation cost. Modern sites which use the new navigation API should never be using `document.open()`.
 
-Second, the following navigations **cannot be canceled** using `event.preventDefault()`, and as such will have `event.cancelable` equal to false:
-
+Second, **traversals have special restrictions on cancelling the navigation** via `event.preventDefault()`. Traversals are:
```

Comment:
Nit: I generally prefer one "l" in cancel-related conjugations, and we've codified this for the WHATWG: https://whatwg.org/style-guide .

Reply:
Done